cookbook/examples/apps/whatsapp_chat_agent/readme.md

```diff
@@ -0,0 +1,94 @@
+# WhatsApp Chat Agent with Stock Market Insights
+
+This is a WhatsApp chatbot that provides stock market insights and financial advice using the WhatsApp Business API. The bot is built using FastAPI and can be run locally using ngrok for development and testing.
+
+## Prerequisites
+
+- Python 3.7+
+- ngrok account (free tier works fine)
+- WhatsApp Business API access
+- Meta Developer account
+- OpenAI API key
+
+## Setup Instructions
+
+1. **Install Dependencies**
+
+```bash
+pip install -r requirements.txt
```

Comment:
Please add the requirements.in and the generate_requirements script like we have with other apps